mlos_bench/mlos_bench/environments/local/local_fileshare_env.py

```diff
@@ -90,7 +90,7 @@ def _template_from_to(self, config_key: str) -> List[Tuple[Template, Template]]:
     def _expand(
         from_to: Iterable[Tuple[Template, Template]],
         params: Mapping[str, TunableValue],
-    ) -> Generator[Tuple[str, str], None, None]:
+    ) -> Generator[Tuple[str, str], None, None]:  # pylint: disable=unnecessary-default-type-args
```

Comment:
These are just R type warnings, and could probably be globally ignored instead of individually.
Python 3.9 doesn't support just removing the arguments though.

Reply:
moved to pyproject.toml global ignore